clustering/marathon.py

```diff
@@ -0,0 +1,686 @@
+    missing = []
+    for parm in OP_REQUIRED[state]:
+        if not module.params[parm]:
+            missing.append(parm)
+    if missing:
+        module.fail_json(msg="Operation %s require the following missing parameters: %s" % (state, ",".join(missing)))
+
+    # Handle rest of parameters
+    uri = module.params['uri']
+    user = module.params['username']
+    passwd = module.params['password']
+
+    # Ensure that we use int values for port mappings
+    if module.params['docker_portMappings']:
+        mappings = module.params['docker_portMappings']
+        mappings = [dict((k, int(v)) for k,v in kv.iteritems()) for kv in mappings]
+        module.params['docker_portMappings'] = mappings
+
+    # Ensure that we use int values for some healthChecks parameters
+    if module.params['healthChecks']:
+        healthChecks = module.params['healthChecks']
+        for checks in healthChecks:
+            for param in ['port', 'gracePeriodSeconds', 'intervalSeconds', 'timeoutSeconds', 'maxConsecutiveFailures']:
+                if param in checks:
+                    checks[param] = int(checks[param])
+        module.params['healthChecks'] = healthChecks
+
+    if module.params['docker_image'] and not module.params['container']:
+        module.params['container'] = { 'type': 'DOCKER', 'docker': { 'image': module.params['docker_image'], 'forcePullImage': bool(module.params['docker_forcePullImage']), 'privileged': bool(module.params['docker_privileged']), 'network': module.params['docker_network'], 'parameters': module.params['docker_parameters'], 'portMappings': module.params['docker_portMappings']}, 'volumes': module.params['container_volumes']}
+    else:
+        if module.params['container_volumes'] and not module.params['container']:
+            module.params['container'] = { 'type': module.params['container_type'], 'volumes': module.params['container_volumes']}
+
+    if module.params['upgradeStrategy_minimumHealthCapacity']:
+        module.params['upgradeStrategy'].update({'minimumHealthCapacity': module.params['upgradeStrategy_minimumHealthCapacity']})
+
+    if module.params['upgradeStrategy_maximumOverCapacity']:
+        module.params['upgradeStrategy'].update({'maximumOverCapacity': module.params['upgradeStrategy_maximumOverCapacity']})
+
+    if not uri.endswith('/'):
+        uri = uri + '/'
+    restbase = uri + 'v2'
+
+    # Dispatch
+    try:
+
+        # Lookup the corresponding method for this operation. This is
+        # safe as the AnsibleModule should remove any unknown operations.
+        thismod = sys.modules[__name__]
+        method = getattr(thismod, state)
+
+        ret = method(restbase, user, passwd, module.params)
+
+    except Exception, e:
+        return module.fail_json(msg=e.message)
+
+
+    module.exit_json(changed=ret.changed, uri=uri, state=state, meta=ret.meta)
+
+
+from ansible.module_utils.basic import *
+from ansible.module_utils.urls import *
+main()
```

Comment:
Could you please update this to:

```
if __name__ == '__main__':
    main()
```

As per http://docs.ansible.com/ansible/developing_modules.html#common-module-boilerplate